pymeasure/instruments/siglenttechnologies/siglent_sdm3045x.py

```diff
@@ -0,0 +1,111 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+from pymeasure.instruments import SCPIMixin, Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+
+class SiglentSDM3045X(SCPIMixin, Instrument):
+    """
+    Driver for the Siglent SDM3045X Digital Multimeter.
+
+    Provides functionality for measuring voltage, current, and temperature.
+    It also allows configuration of DC voltage range, filter settings,
+    and switching between 2-wire and 4-wire measurement modes.
+    """
+
+    def __init__(self, adapter, name="Siglent SDM3045X Multimeter", **kwargs):
+        """
+        Initializes the instrument with the given VISA adapter.
+
+        Args:
+            adapter: VISA Adapter used to communicate with the instrument.
+            name: Optional name for the instrument (default: "Siglent SDM3045X Multimeter").
+        """
+        super().__init__(
+            adapter,
+            name,
+            **kwargs
+        )
+
+    # Measurement Modes
+    measurement_mode = Instrument.control(
+        "CONFigure?",
+        "CONFigure:%s",
+        "Set the measurement mode "
```

Comment:
If it is a `control`, please use "Control:
```suggestion
        "Control the measurement mode "
```

Set is for a `setting` and Get/Measure for a `measurement`.